package.json

```diff
@@ -59,6 +59,23 @@
             }
           }
         },
+        "githubPullRequests.pullRequestName": {
+          "type": "string",
+          "enum": [
+            "commit",
+            "branch",
+            "custom",
+            "ask"
+          ],
+          "enumDescriptions": [
+            "Use the latest commit message",
+            "Use the branch name",
+            "Specify a custom message",
+            "Ask which of the above methods to use"
+          ],
+          "default": "ask",
+          "description": "The name used when creating pull requests."
```

Comment:
Let's use "title" instead of "name" here to be consistent with the GitHub UI. Ditto for setting name